src/funcs.c

```diff
@@ -1409,6 +1452,7 @@ void game_properties(void)
 		FreeMem(fullpath, 800 * sizeof(char));
 	if (str2)
 		FreeMem(str2, 512 * sizeof(char));
+#endif
```

Comment:
Maybe it would be better to free memory in a separated method, since there is a check for the system, like I mentioned above  for setting  a memory area.